pkg/agent/run.go

```diff
@@ -40,6 +42,9 @@ var OutputPath string
 // InputPath is where the agent will read data from instead of gathering from clusters if specified
 var InputPath string
 
+// BackoffMaxTime is the maximum time for which data gatherers will be retried
+var BackoffMaxTime uint
```

Comment:
Just an idea, but this could be a `time.Duration` so that it can be used wherever without worrying about converting it, and there's no ambiguity as to the units.

Reply:
The same has been done for the `period` command line argument. I'll update that too